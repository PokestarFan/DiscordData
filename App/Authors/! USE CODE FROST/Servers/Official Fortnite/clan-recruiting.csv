ID,Content Length,Content
560623578978975745,22,wanna join a team nae.
