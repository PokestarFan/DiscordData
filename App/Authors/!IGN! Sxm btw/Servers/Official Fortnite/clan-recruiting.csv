ID,Content Length,Content
558784696746770445,24,who grinding for chronic
