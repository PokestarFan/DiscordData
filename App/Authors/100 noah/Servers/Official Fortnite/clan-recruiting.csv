ID,Content Length,Content
560650365083582484,140,18 clan recruiting! Any servers and any platform! These are the requirements! No being toxic in clan! Must be a good builder! DM me to join!
