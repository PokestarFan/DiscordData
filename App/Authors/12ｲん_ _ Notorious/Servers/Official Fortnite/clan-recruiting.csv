ID,Content Length,Content
558467899325677568,51,Looking for PC PLAYERS to lead a team for me EU 18+
560640663205445653,43,Any teams need banners and logos_ hit me up
