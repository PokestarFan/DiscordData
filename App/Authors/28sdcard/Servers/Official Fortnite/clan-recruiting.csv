ID,Content Length,Content
558773049084870694,4,asdf
