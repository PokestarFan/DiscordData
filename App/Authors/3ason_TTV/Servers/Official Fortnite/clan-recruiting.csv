ID,Content Length,Content
558867408421388308,43,Team Asynic is not trying out DM for tryout
