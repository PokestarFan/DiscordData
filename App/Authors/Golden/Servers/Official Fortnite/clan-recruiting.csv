ID,Content Length,Content
560998207728123930,807,Dark eSports _ we are a new clan We accept any platform                                                                                                                                                  Causal Team _ Wins 10+NL Professional Team _ 50 Wins +NL Legends Team _ 100 Wins +                                                                                                                                                                                                                         DM If For Invite Link 👌NLplease advertiseNLif you canNLthanks!NLRequirements ❗ NLNL👉 MicrophoneNL👉 Able To Speak EnglishNL👉 Active On Fortnite/DiscordNL👉 Age 13+ NLNL>>>Attitude<<<NLNL✴ PoliteNL✴ PositiveNL✴ MatureNL✴ DiscordNLNL ❕  Teams ❕ NL👉 PC Both NA And EUNL👉 PlayStation All RegionsNL👉 Xbox All RegionsNL👉 Switch All Regions
